build/cmake/package_tester/modules/tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+declare -A test_start_state
+declare -A test_exit_state
+declare -a tests
+
+if [ -z "${tests_sh_included}" ]
+then
+   tests_sh_included=1
+
+   source ${source_dir}/modules/util.sh
+   source ${source_dir}/modules/testing.sh
+
+   tests=( "fresh_install" "keep_config" )
+   test_start_state=([fresh_install]=INSTALLED [keep_config]=CLEAN )
+
+   fresh_install() {
+       tests_healthy
+       success Fresh installation is not clean
+       # test that we can read from and write to fdb
+       cd /
+       fdbcli --exec 'writemode on ; set foo bar'
+       success Cannot write to database
+       getresult="$(fdbcli --exec 'get foo')"
+       success Get on database failed
```

Comment:
Nit: maybe `successOr` would be a better name? I don't have a strong opinion here.

Reply:
I agree, I renamed the function